tests/unit/python/fledge/common/test_configuration_manager.py

```diff
@@ -1,17 +1,17 @@
 # -*- coding: utf-8 -*-
 
 import asyncio
-import json
 import ipaddress
-from unittest.mock import MagicMock, patch, call
+import json
 import pytest
 
-
-from fledge.common.configuration_manager import ConfigurationManager, ConfigurationManagerSingleton, _valid_type_strings, _logger, _optional_items
+from unittest.mock import MagicMock, patch, call
```

Comment:
standard import "from unittest.mock import MagicMock, patch, call" should be placed before "import pytest" (wrong-import-order)